components/Operations.js

```diff
@@ -88,8 +91,9 @@ function Operation({ type, asyncapi, operation, channelName, channel }) { // NOS
             <ListItem>
               Available only on servers:{' '}
               {servers.map(s => {
-                const slug = FormatHelpers.slugify(s);
-                return `[${s}](#${slug}-server)`;
+                const serverId = s.id();
```

Comment:
tested with https://raw.githubusercontent.com/asyncapi/asyncapi/2.0.0/examples/2.0.0/streetlights.yml

with current template release, it is now showing up, as I do not have any info on a channel level that given channel is only or `production` or somewhere else

with this PR, I get text: `	* Available only on servers: [production](#production-server)` which is not true

Reply:
Technically it's true because, without the channel explicitly defining a server, it's every server defined in servers.

Reply:
This is the parser-api's doing, because it defaults to all servers 🙂

Reply:
Question is, do you want it to be the same as before or with this new display?

Reply:
but there is just one server, so sentence `Available only on servers: production` is only confusing, like `wait are there any other servers available?`. It's like with a warning on a plastic bag to not put it on your head, like `why would I even try` 😄

Reply:
They do it in the US 😆

Reply:
yeah, I know 🤣 but markdown-template is not used only in the US 😄

Reply:
Alright, changed the implementation to only show the servers the application is operating on when it is NOT all servers 🙂